contributed_definitions/nyaml/NXsample_surface_dopant.yaml

```diff
@@ -0,0 +1,13 @@
+category: base
+doc: |
+  Description of a dopant material embedded in a sample surface.
+type: group
+NXsample_surface_dopant(NXsample_component):
+  element:
+    doc: |
+      Element of evaporated surface dopant such as alkali or other
+  coverage:
+    unit: NX_LENGTH
```

Comment:
I would also call this then thickness or something. Coverage sounds like a fractional surface coverage.